ddtrace/_trace/trace_handlers.py

```diff
@@ -691,6 +691,28 @@ def _on_botocore_patched_bedrock_api_call_success(ctx, reqid, latency, input_tok
     span.set_tag_str("bedrock.usage.completion_tokens", output_token_count)
 
 
+def _on_rq_queue_enqueue_job_is_async(ctx, job):
+    span = ctx[ctx["call_key"]]
+    HTTPPropagator.inject(span.context, job.meta)
+
+
+def _on_rq_after_perform_job(ctx, job):
```

Comment:
We can make this function more general. It can accept a list of key value pairs and an isfailed flag that is used to determine whether span.error=1 should be set. This will allow us to reuse this method in other integrations

Reply:
@mabdinur hopefully i succeeded in this?

Reply:
Yup this looks awesome!